pkg/distributor/distributor.go

```diff
@@ -327,6 +327,7 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 			Help:      "Number of labels per sample.",
 			Buckets:   []float64{5, 10, 15, 20, 25},
 		}),
+		// TODO switch to native histogram eventually and add "type" label
```

Comment:
We prefer to create issues instead of TODOs.

Switching existing metrics native histograms too requires that people using Mimir have their monitoring ready for scraping, storing and querying native histograms. I think we should instead introduce new histogram metrics in addition to existing ones, in order to avoid breaking people's monitoring. If we want, we can do that today (in separate PR).

Reply:
Moved to issue, removed TODOs https://github.com/grafana/mimir/issues/4175